swagger.yaml

```diff
@@ -31,6 +45,23 @@ paths:
             $ref: '#/definitions/RecipeList'
         500:
           $ref: '#/responses/InternalError'
+    post:
```

Comment:
I think there should be more new routes added? the /recipes/id/instances* all of those should be documented

Reply:
Done in commit "update swagger spec with new added routes"